### PR TITLE
Pass arguments in to ammonite

### PR DESCRIPTION
**Pull request by pismute**
Hi, 

Thank you for the great discussion on #1148, I made an update.

Not only skipping global `predef.sc`, user can use local `predef.sc` like:

```
bloop console $PROJECT_NAME -- --predef=$PROJECT_DIR/predef.sc 
```

In terms of a project, default `--no-home-predef` makes sense, but I just took a simple solution.

### CODE REVIEW

**Comment by maxstreese** [comment]
Thanks for implementing the suggested changes @pismute! I will close my PR in favor of yours :smiley:

File: frontend/src/main/scala/bloop/cli/Commands.scala [hunk 1]
```diff
@@ -163,6 +163,8 @@ object Commands {
         s"Pick REPL to run console. The default is Ammonite, available REPLs are: ${ReplKind.repls.map(_.name).mkString(", ")}"
       )
       repl: ReplKind = AmmoniteRepl,
+      @HelpMessage("The arguments to pass in to Ammonite.")
+      args: List[String] = Nil,
```
[the remark this answers is not in the document]
Comment (pismute) [reply frontend/src/main/scala/bloop/cli/Commands.scala]: Within current interfaces, yes. There are no way to pass `extraArgs` to `bloop.engine.Interpreter` from `bloop.Cli`

IMO If we want to remove `--args` option, we could add it to `Run` like:

```
case class bloop.engine.Run(command: Commands.Command, next: Action, extraArgs: Seq[String]) extends Action
```

`bloop test` and `bloop run` also handles it in the same way. To be honest, I did `copy and paste`.

**Comment by pismute** [comment]
Hi @jvican, I can't find why it is failed, what should I look into?

File: frontend/src/main/scala/bloop/cli/Commands.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (jvican) [reply frontend/src/main/scala/bloop/cli/Commands.scala]: Oh I see why this exists, you're using `extraArgs` to populate it in here. If it's the same as in `run` and `test` I'm fine doing this :+1:

**Comment by jvican** [comment]
@pismute I've just fixed it by running the command that the CI error proposed. Merging, the only single error we have in the CI is spurious. Thanks for the contribution!

**Pull request merged**